### PR TITLE
5.0.1 updates

### PR DESCRIPTION
**Pull request by nadinequinr3**
Preview: https://nadine.preview.docs.r3.com/en/platform/corda/5.0.html

### CODE REVIEW

File: content/en/platform/corda/5.0/deploying-operating/deployment/deploying/_index.md [hunk 1]
```diff
@@ -122,14 +78,12 @@ The following sections describe how to download the Corda {{< tooltip >}}Helm{{<
 If you have access to Docker Hub, you can download the Corda Helm chart using the following command:
 
 ```shell
-helm fetch oci://registry-1.docker.io/corda/corda --version {{<version-num>}}.0
+helm fetch oci://registry-1.docker.io/corda/corda --version 5.0.1.0
```
Comment (ronanbrowne) [inline content/en/platform/corda/5.0/deploying-operating/deployment/deploying/_index.md:81]:
```suggestion
helm fetch oci://registry-1.docker.io/corda/corda --version 5.0.1
```

helm charts just use 3 digits

**Review by ronanbrowne — CHANGES_REQUESTED** [review]
See comment

**Review by ronanbrowne — APPROVED** [review]
LGTM

File: content/en/platform/corda/5.0/deploying-operating/deployment/deploying/_index.md [hunk 2]
```diff
@@ -27,59 +28,14 @@ This section contains the following:
...
-   target_registry=$1
 
-   for image in "${images[@]}"; do
-    source=corda/$image:$tag
-    target=$target_registry/$image:$tag
-    echo "Publishing image $source to $target"
-    docker tag $source $target
-    docker push $target
-   done
+{{< enterprise-icon noMargin="true" >}} If you are deploying Corda Enterprise, you can push the images from the provided `tar` file into a container registry that is accessible from the cluster:
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.0/deploying-operating/deployment/deploying/_index.md:32]:
```suggestion
{{< enterprise-icon noMargin="true" >}} If you are deploying Corda Enterprise, you can push the images from the provided `.tar` file into a container registry that is accessible from the cluster:
```

**Review by AnnaK-R3 — APPROVED** [review]

**Pull request merged**